meteor/sofie-server/model/value-objects/part-timings.ts

```diff
@@ -0,0 +1,6 @@
+export interface PartTimings {
+	inTransitionStart: number | null // The start time within the toPartGroup of the inTransition
```

Comment:
We should consider whether or not to use null. There is a slight difference in the semantics of `undefined` and `null`, but in most cases `undefined` should suffice. It also reduces the number of empty/invalid value cases we should handle. This could perhaps be changed to:
```suggestion
	inTransitionStart: number | undefined // The start time within the toPartGroup of the inTransition
```

Reply:
I agree we should stick to only using undefined. Only reason I used null is because that is what is used for the interface by Core. Will update it

Reply:
Updated